ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/config/SandboxConfig.scala

```diff
@@ -59,7 +59,7 @@ object SandboxConfig {
       port = DefaultPort,
       portFile = None,
       damlPackages = Nil,
-      timeProviderType = None,
+      timeProviderType = Some(TimeProviderType.WallClock),
```

Comment:
Why not make this a `TimeProviderType` rather than an `Option[TimeProviderType]`?

Reply:
Because this allows us to detect when users provide both `-s` *and* `-w`. See https://github.com/digital-asset/daml/blob/5cd3311a900656973008c81829f6257026a1bd3b/ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/cli/Cli.scala#L259-L267

Reply:
Right, but you're setting the default now, which means you won't be able to specify `--static-time` at all. (This is why your tests are failing.)

I think you want to set it back to `None` and make sure we use `.getOrElse(WallClock)`.

Reply:
Yeah, thanks. This is why a bunch of tests failed. I've set it back to None. 👍